package.json

```diff
@@ -9,17 +9,20 @@
     "@testing-library/user-event": "^12.1.10",
     "graphql": "^15.4.0",
     "graphql-query-builder": "^1.0.7",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "react-scripts": "4.0.1",
+    "react": "^16.13.0",
```

Comment:
@risenW why are we downgrading here? Is it Storybook requirement?

Reply:
Yea, it's a storybook requirement. Ref https://www.learnstorybook.com/intro-to-storybook/react/en/get-started/